src/clj/pod/epiccastle/bbssh/pod/session.clj

```diff
@@ -21,6 +21,19 @@
    ^Session (references/get-instance session)
    ^UserInfo (references/get-instance user-info)))
 
+(defn make-proxy
+  [{:keys [type host port]}]
```

Comment:
would be good to take :username and :password options, too. https://github.com/mwiede/jsch/blob/b3e4481583934e5e3ed40ce7bd43d3c4f95e6cda/src/main/java/com/jcraft/jsch/ProxySOCKS5.java#L69